docker/README.md

```diff
@@ -3,7 +3,7 @@ in the environment file.  Its value will be prepended along with the service
 name to the container on start up.
 
 
-You may use multiple compose files to customize your container.  For instance, you
+You may use multiple compose files to customize your container. For instance, you
```

Comment:
Why did you change?  This is likely me using two spaces between sentences.
In any case, not really related to this PR.  I am not requesting changes but if there is any further push I would like you to undo.

Reply:
I changed it in [a9dba00](https://github.com/arborx/ArborX/commit/a9dba003b35ec6c8ba2ca155ac81a45dd085d791), before your review, when I needed to force push to fix the incorrect Jenkins links, and was looking for a minor change.